includes/CLI/Plugin_Check_Command.php

```diff
@@ -242,7 +400,7 @@ static function ( $dirs ) use ( $excluded_directories ) {
 	 *
 	 * @throws WP_CLI\ExitException Show error if plugin not found.
 	 */
-	private function get_options( $assoc_args ) {
+	private function get_check_options( $assoc_args ) {
```

Comment:
Now that I see how similar the three `get_*_options()` methods are (like you already pointed out in https://github.com/WordPress/plugin-check/pull/339#issuecomment-1875228573), I think we should consolidate them to avoid duplicate logic.

Two options I can think of:
* Either we move the `$options['format']` check that is in each of the 3 methods into a separate private method and call it from all places. Benefit: Keeps the code decoupled and allows to make individual changes to each of the three checks.
* Or we remove the 3 methods in favor of only keeping a single `get_options()` method. Then that method would need a new `$defaults` parameter (since that is the only difference between the 3 methods). Benefit: Even less duplicate code.

Let me know what you think.

Reply:
I have used second approach.